oxenss/storage/database.cpp

```diff
@@ -287,6 +287,30 @@ ALTER TABLE messages ADD COLUMN namespace INTEGER NOT NULL DEFAULT 0;
             )");
         }
 
+        if (!db.tableExists("revoked_subkeys")) {
+            log::info(logcat, "Upgrading database schema");
+            db.exec(R"(
+CREATE TABLE revoked_subkeys (
+    id INTEGER,
+    owner INTEGER REFERENCES owners(id) ON DELETE CASCADE,
+    subkey TEXT NOT NULL,
```

Comment:
I think this should be a `BLOB` rather than `TEXT`; if it's text we'll have to convert it to something "safe" (e.g. hex) but it's probably easier just to leave it as binary everywhere.